.github/workflows/docs.yml

```diff
@@ -5,7 +5,7 @@ on:
   push:
     branches:
       - main
-    path:
+    paths:
```

Comment:
The rustdoc-generated docs also change any time the Rust code changes, can we loosen this trigger restriction? This job doesn't take long // @gustavovalverde

Reply:
@dconnolly did you want to open a ticket to get this fixed?

Reply:
I opened https://github.com/ZcashFoundation/zebra/issues/3534